### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false positives for `word-break: auto-phrase`

### ISSUE THREAD

**Issue by Aquaticat** [stylelint/stylelint#7939]
### What minimal example or steps are needed to reproduce the bug?

```css
.foo {
  word-break: auto-phrase;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "declaration-property-value-no-unknown": true
  }
}
```

### How did you run Stylelint?

[word-break: auto phrase - Demo | Stylelint](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0AzCEAEwA6A7bALgO4QBOUAtAEakwCGA1vBnQK4EQUAOAFqXQGcYAbjwBfPCAA0IFAEsANjABydALZxEMAB7quSpGAEDp4CLnkBzBCBy4MGbCFKslAp8zsOHT2GAV0-ARy5tykEFwwpAQAnhQAbnQKrDAUuJysuAzpRPgIGAQuMHgOErhippAWcpYAYmRqdAQ2AFYC5qawXCaIXo4gArFKCnK4BB79AQQwg05SJf2DMcOjBBRVVhSDdLhQgVATTlMz4yDiIGJAA)

### Which Stylelint-related dependencies are you using?

stylelint 16.8.2 locally, and in the Stylelint demo, 16.8.2 too.

### What did you expect to happen?

No problems to be reported.

### What actually happened?

The following problems were reported: 

Unexpected unknown value "auto-phrase" for property "word-break"
[(declaration-property-value-no-unknown)](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown)

### More information

On [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), it's listed as experimental.

**Comment by Mouvedia** [comment]
Technically, it can be fixed on our side but it ought to be fixed on `css-tree`'s side.
i.e. the status is either _ready to implement_ or _blocked_

**Comment by Aquaticat** [comment]
Thanks, I just made a issue there: <https://github.com/csstree/csstree/issues/277>
